packages/react/src/components/DataTable/components/DataTableColumn/DataTableColumn.tsx

```diff
@@ -1,10 +1,23 @@
 import * as React from 'react';
 import { ChevronDown, ChevronUp, Sort } from '@project44-manifest/react-icons';
 import { cx } from '@project44-manifest/react-styles';
-import { flexRender, RowData } from '@tanstack/react-table';
+import {
+  AccessorColumnDef,
+  DisplayColumnDef,
+  flexRender,
+  GroupColumnDef,
+  RowData,
+} from '@tanstack/react-table';
 import { getColumnLayoutStyles } from '../../utils';
 import { DataTableColumnProps } from './DataTableColumn.types';
 
+export interface ExtendedColumnDef<TData extends RowData, TValue = unknown>
+  extends Omit<DisplayColumnDef<TData, TValue>, 'isGroupStart'>,
+    Omit<GroupColumnDef<TData, TValue>, 'isGroupStart'>,
+    Omit<AccessorColumnDef<TData, TValue>, 'isGroupStart'> {
+  isGroupStart?: boolean;
```

Comment:
looks so complicated to me. 
won't it work without Omit?

Reply:
@vitaliy-p44 Since these types are exposed from tan-stack and we don't have easy way to extend them, that's why i have used this. I tried multiple things and none seems to work except this.

If you have any suggestion do let me know, i can try that as well.